src/Data/Impact.elm

```diff
@@ -507,81 +506,6 @@ computeAggregatedScore definitions getter (Impacts impacts) =
         |> Definition.foldl (\_ -> Quantity.plus) Quantity.zero
 
 
-minEcotoxWeighting : Unit.Ratio
-minEcotoxWeighting =
-    Unit.ratio 0
-
-
-maxEcotoxWeighting : Unit.Ratio
-maxEcotoxWeighting =
-    Unit.ratio 0.25
-
-
-{-| Set the ecotoxicity weighting (EtfC) then redistribute other Ecoscore weightings
-accordingly. The methodology and formulas are described in this card:
-<https://www.notion.so/Rendre-param-trable-la-pond-ration-de-l-cotox-894d42e217c6448a883346203dff8db4>
-FIXME: ensure the card contents are moved to the public documentation eventually
--}
-setEcotoxWeighting : Unit.Ratio -> Definitions -> Definitions
-setEcotoxWeighting (Unit.Ratio weighting) definitions =
-    let
-        defsToUpdate =
-            [ Definition.Acd
-            , Definition.Fru
-            , Definition.Fwe
```

Comment:
Note to reviewers: this is obsolete dead code which was only used by… unit tests.